galleries/examples/images_contours_and_fields/image_demo.py

```diff
@@ -30,7 +30,7 @@
 Z = (Z1 - Z2) * 2
 
 fig, ax = plt.subplots()
-im = ax.imshow(Z, interpolation='bilinear', cmap=cm.RdYlGn,
```

Comment:
Is there a reason we should not use one of the perceptually uniform colormaps here?

Reply:
Chose a different diverging colormap.

Reply:
@tacaswell the mapping is centered on 0. It makes sense to use a divergent colormap to distinguish positive and negative values.